cheatsheets/gleam-for-python-users.md

```diff
@@ -499,11 +500,14 @@ Case is one of the most used control flow in Gleam. It can be seen as a switch s
 
 #### Python
 
-Python does not have switch statements. `if/else` statements paired to the `type` function let the developer to write code that depends on the type and value of a variable.
-
-Until now, there has been no official comparable feature for the case expression used in Gleam.
+Before the release of Python 3.10 there hasn't been a comparable feature for
+the case expression used in Gleam. `if/else` statements paired to the `type`
+function let the developer write code that depended on the type and value of a
+variable.
 
-However this will change in Python 3.10 (currently in [alpha release]([https://docs.python.org/3.10/whatsnew/3.10.html#pep-634-structural-pattern-matching]), at the time of writing) that ships structural pattern matching allowing:
+However this changed in Python 3.10 when [structural pattern
+matching]([https://docs.python.org/3.10/whatsnew/3.10.html#pep-634-structural-pattern-matching])
+was introduced. This allows:
```

Comment:
This is a whole lot of words to say "Python 3.10 added [structural pattern matching], which is comparable to Gleam's `case` expression" 😅

Reply:
I added a similar sentence like you suggest and moved it directly under the 'Case' header. That seemed most in line with how it is done in other sections. Now the Python and Gleam sections just show examples of how it's done in either language.

Do you think it would be helpful if the examples would be more similar or is that not really needed?